src/core/public/capabilities/capabilities_service.mock.ts

```diff
@@ -16,33 +16,30 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import { CapabilitiesService, CapabilitiesSetup, UICapabilities } from './capabilities_service';
```

Comment:
should it have `UI-` prefix?

Reply:
I debated this. The legacy implementation refers to this as `UICapabilities`, but we're just calling them `Capabilities` in the new platform. I'll rename to Capabilities now, but shim it in legacy world to prevent this PR from blowing up too much with a global rename.